test/hotspot/jtreg/serviceability/jvmti/events/NotifyFramePopStressTest/NotifyFramePopStressTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @summary Verifies NotifyFramePop request is cleared if JVMTI_EVENT_FRAME_POP is disabled
+ * @requires vm.jvmti
+ * @library /test/lib
+ * @compile NotifyFramePopStressTest.java
+ * @run main/othervm/native -agentlib:NotifyFramePopStressTest NotifyFramePopStressTest
+ */
+
+import jtreg.SkippedException;
+
+public class NotifyFramePopStressTest {
+    static volatile boolean done = false;
+    static volatile int notifyCount = 0;
+
+    static {
+        try {
+            System.loadLibrary("NotifyFramePopStressTest");
+        } catch (UnsatisfiedLinkError ex) {
+            System.err.println("Could not load NotifyFramePopStressTest library");
+            System.err.println("java.library.path:"
+                + System.getProperty("java.library.path"));
+            throw ex;
+        }
+    }
+
+    public static void main(String args[]) {
+        if (!canGenerateFramePopEvents()) {
+            throw new SkippedException("FramePop event is not supported");
+        }
+        Thread testThread = Thread.currentThread();
+        Thread controlThread = new Thread(() -> control(testThread), "Control Thread");
+
+        setFramePopNotificationMode(testThread, true);
```

Comment:
I think we can get rid of this API and just have the native code default to enabling FRAME_POP events. This was copied from a test that would turn it on and off, but this test always needs it on.

Reply:
Thank you for the suggestion.
I've removed the second parameter and defaulted it to `true`.
The API is still useful as it points to the tested thread the FramePop events have to be enabled for.